Packs/CommonScripts/Scripts/displayUtilitiesResults/displayUtilitiesResults.yml

```diff
@@ -0,0 +1,18 @@
+commonfields:
+  id: displayUtilitiesResults
```

Comment:
should be in XSOAR marketplace? or only on XSOAR?

Reply:
Can benefit both